pkg/make-github-release.sh

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env bash
```

Comment:
Deleting the release through the Github CLI sure sounds easier, I'll give that a try. I don't like the release upload action too much though because it doesn't label the artifacts properly, you just get raw file name gibberish that a mortal can't tell apart. That too can be replaced with a `gh release create ...` though apparently.

Reply:
My experience has been that a significant number of people will see a pile of links that are "the same" followed by two links that say "code". Okay, this is clearly techy tech stuff and there's even coding involved somehow, better ask which buttons they're supposed to press.

Even if they manage to spot that the file extensions are distinct, they may not recognize what's what from that alone. I don't have first-hand painful experiences with that, since I've only been uploading zips, but I could very well imagine that ".msi" is not a particularly well-known format (most installers seem to be .exe) and translating ".zip" to "portable Windows version" is quite a leap, especially given that right below it there's another zip option for source code.

Replacing the bulk of the script with two tiny Github CLI calls has worked anyway, I think that solution is reasonably short while giving a result with pretty names.